### PR TITLE
GNOME User Guide Update

### PR DESCRIPTION
**Pull request by taroth21**
### Description
Updating screenshots and addressing Jira issues for SP2 (SLE-8685, SLE-8167).

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/gnome_custom.xml [hunk 1]
```diff
@@ -133,17 +121,15 @@
...
      </para>
     </listitem>
     <listitem>
      <para>
       <guimenu>Formats</guimenu>: To change the formats for date, number,
-      currency and related options, choose a country from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      currency and related options, choose a country from the list. To make the
+      change take effect, you need to restart the desktop afterwards.
```
Comment (lproven) [inline xml/gnome_custom.xml:132]: Can we clarify "restart the desktop"? Does it mean log out and log back in, or is a reboot necessary?

File: xml/gnome_custom.xml [hunk 2]
```diff
@@ -271,33 +257,34 @@
...
+        make changes to your compose file, you must log out of your session and
         log back in to activate the changes.
     </para>
     <para>
-        Your key assignments are case-sensitive. 
+        Your key assignments are case-sensitive.
         <filename>~/.XCompose</filename> overrides the system files. Use the 
         compose file for your locale to see what is already configured, and to
         copy symbols from it into your personal compose file. (These files
         may have a lot of whitespace at the beginning; they're not empty so keep
```
Comment (lproven) [inline xml/gnome_custom.xml:274]:
```suggestion
        may have a lot of whitespace at the beginning; they are not empty so keep 
```

File: xml/gnome_custom.xml [hunk 3]
```diff
@@ -581,7 +553,7 @@
 
   <sect2 xml:id="sec-gnome-settings-hardware-screen-multi">
    <title>Changing Settings: Multiple Monitor Setup</title>
-
+<!--taroth 2020-04-09: could not check mutli-display setup on the VM-->
```
Comment (lproven) [inline xml/gnome_custom.xml:556]:
```suggestion
<!--taroth 2020-04-09: could not check multi-display setup on the VM-->
```

Comment (lproven) [reply xml/gnome_custom.xml:556]: I know it is just a comment. :-)

File: xml/gnome_custom.xml (hunk 1 above)
Comment (taroth21) [reply xml/gnome_custom.xml:132]: I just checked: Obviously one just needs to restart the desktop session and GNOME prompts you to do so. If you accept, you will be logged out the desktop and have to log back in.

File: xml/gnome_custom.xml [hunk 4]
```diff
@@ -133,17 +121,16 @@
    <itemizedlist>
     <listitem>
      <para>
-      <guimenu>Language</guimenu>. Choose a language from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      <guimenu>Language</guimenu>. Choose a language from the list. To make the
+      change take effect, you will be prompted to restart the desktop session
```
Comment (lproven) [inline xml/gnome_custom.xml:125]: I suggest that the wording at line 267:

"you must log out of your session and log back in to activate the changes."

... is clearer. So we could say:

"You will be prompted to [literal]Restart the session for changes to take effect[/literal]. You must log out of your session and log back in (or reboot) to do so."

The (or reboot) is optional. It is possibly obvious enough not to mention.

File: xml/gnome_custom.xml [hunk 5]
```diff
@@ -133,17 +121,16 @@
...
      </para>
     </listitem>
     <listitem>
      <para>
       <guimenu>Formats</guimenu>: To change the formats for date, number,
-      currency and related options, choose a country from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      currency and related options, choose a country from the list. To make the
+      change take effect, you need to restart the desktop afterwards.
```
Comment (lproven) [inline xml/gnome_custom.xml:133]: As per previous comment, we could say:

"To make the changes take effect, you must log out of your session and log back in."

**Pull request merged**